dask/tests/test_tokenize.py

```diff
@@ -185,12 +161,11 @@ def test_tokenize_numpy_array_on_object_dtype():
         np.array([(1, "a"), (1, None), (1, "aaa")], dtype=object)
     ) == check_tokenize(np.array([(1, "a"), (1, None), (1, "aaa")], dtype=object))
 
-    # Trigger non-deterministic hashing for object dtype
-    class NoPickle:
+    class NeedsCloudPickle:
         pass
 
-    x = np.array(["a", None, NoPickle], dtype=object)
-    check_tokenize(x, idempotent=False)
+    x = np.array(["a", None, NeedsCloudPickle], dtype=object)
```

Comment:
Probably doesn't matter much but it's more likely to be an instance
```suggestion
    x = np.array(["a", None, NeedsCloudPickle()], dtype=object)
```